plugins/mcreator-localization/help/default/entity/water_entity.md

```diff
@@ -1,3 +1,3 @@
 Checking this parameter makes the entity a water entity.
 
-Enabling this parameter will make the entity have water type navigator and motion controller.
\ No newline at end of file
+Enabling this parameter will make the entity have a water-type navigator, and water motion controller and spawn in fluids.
```

Comment:
This parameter alone will not make entity spawn in fluid, it will only allow spawning of entity in fluids (by default fluid is not "can spawn" region for entities), which is different.

Reply:
Oh ok, I didn't understand correctly the first time.